packages/babel-parser/test/fixtures/experimental/await-operations/invalid-non-async-context-2/input.js

```diff
@@ -0,0 +1,4 @@
+async () => {
+  function f () { await.all [] };
```

Comment:
It would be good if all these tests used `await.all [x]`, to show that it's still properly parsed as a member expression. Otherwise, LGTM :+1: